### PR TITLE
OEUI-70: Create actions to add drug orders

### PR DESCRIPTION
**Pull request by Betty-Kebenei**
# **JIRA TICKET NAME:**

[OEUI-70: Create actions to add drug orders](https://issues.openmrs.org/projects/OEUI/issues/OEUI-70)

## **SUMMARY:**
This involves making post requests to `/encounter` endpoint, in order to add drug orders.

### CODE REVIEW

**Comment by djazayeri** [comment]
I just peeked at this PR (and I didn't look closely) but I want to make sure that the approach being taken is that the user can create multiple drug orders in the UI, and they are all persisted at once when they click Save.

File: app/js/actions/addDrugOrder.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+});
+export const postDrugOrder = (ordersPayload, patientUuid, careSetting) => (dispatch) => {
+  dispatch(postDrugOrderLoading(true));
+  return axios.post(`${apiBaseUrl}/encounter`, ordersPayload)
+    .then(() => {
+      dispatch(postDrugOrderSuccess(), () => {
+        dispatch(fetchCurrentSession());
+        dispatch(fetchPatientNote(patientUuid));
+        // dispatch(activeOrderAction(patientUuid, careSetting));
+        dispatch(getPastOrders(patientUuid, careSetting));
```
[the remark this answers is not in the document]
Comment (Betty-Kebenei) [reply app/js/actions/addDrugOrder.js]: As you can see the dispatch for active orders is commented, but is there. Reason for commenting is that the PR for active orders is not yet merged. Once it is merged then I will uncomment that line.

[the remark this answers is not in the document]
Comment (Betty-Kebenei) [reply app/js/actions/addDrugOrder.js]: The reason why I dispatch the getPastOrders is that this same action will be used to make edits to an order. When making edits, the previous version is moved to the past orders while the new version appears under active orders.

**Comment by dkayiwa** [comment]
Some merge conflicts here.

**Pull request merged**